sssom/writers.py

```diff
@@ -328,7 +328,7 @@ def to_json(msdf: MappingSetDataFrame) -> JsonObj:
 # Support methods
 
 
-def get_writer_function(output_format, output):
+def get_writer_function(*, output_format: Optional[str] = None, output: str):
```

Comment:
using `*` to make all arguments keyword only is a pretty good way to force that you write code that looks more consistent (e.g., the name of the variable that's passed around should probably be the same, and not change from function to function)

Reply:
I have never seen this, can you link some docs that explain this?

Reply:
https://www.python.org/dev/peps/pep-3102/ gave the spec, this is a short good blog post explaining it for humans https://lukeplant.me.uk/blog/posts/keyword-only-arguments-in-python/

Reply:
Wow! Never heard of this. Thanks